services/rds/src/test/java/software/amazon/awssdk/services/rds/PresignRequestHandlerTest.java

```diff
@@ -141,7 +141,7 @@ public void testSourceRegionRemovedFromOriginalRequest() throws URISyntaxExcepti
     private SdkHttpFullRequest marshallRequest(CopyDBSnapshotRequest request) throws URISyntaxException {
         SdkHttpFullRequest marshalled = SdkHttpFullRequestAdapter.toHttpFullRequest(marshaller.marshall(request));
         return marshalled.toBuilder()
-                         .endpoint(new DefaultServiceEndpointBuilder(RDSClient.SERVICE_NAME, Protocol.HTTPS.toString())
+                         .endpoint(new DefaultServiceEndpointBuilder("rds", Protocol.HTTPS.toString())
```

Comment:
SERVICE_NAME was renamed to SIGNING_NAME.

I could re-add a concept of SERVICE_NAME but not sure where it would be used.